torch/csrc/jit/passes/bailout_graph.cpp

```diff
@@ -154,11 +154,12 @@ struct BailOutInserter {
   // any given bailout point
   void addUnoptimizedFuncToBailouts() {
     auto unoptimized_graph = graph_->copy();
-    auto func = std::make_shared<Function>(
-        std::string{"bailout"}, false, unoptimized_graph, nullptr);
-    auto unopt_func = graph_->create(prim::Constant);
-    unopt_func->output()->setType(FunctionType::create(std::move(func)));
-    unopt_func->insertBefore(*graph_->block()->nodes().begin());
+    auto unopt_func =
```

Comment:
If this is planned to be in there for the long term (which seems reasonable because the original bailout graph is never executed directly so does not need to be a function), then we should give a name like prim::BailoutTemplate so it is more clear where it is being used.